index.js

```diff
@@ -0,0 +1,179 @@
+import './style.css';
+import setState from './modules/getStates.js';
+
+let tasks = [];
+const taskWrapper = document.querySelector('.to-dos');
+const newTask = document.querySelector('.new-task');
+const addNewTask = document.querySelector('.submit');
+const clearAll = document.querySelector('.clear-all');
+
+const addToLocalStorage = () => {
+  localStorage.setItem('myTasks', JSON.stringify(tasks));
+};
+
+const getFromLocalStorage = () => {
+  if (localStorage.getItem('myTasks')) {
+    tasks = JSON.parse(localStorage.getItem('myTasks'));
+  }
+  return tasks;
+};
+
+const resetIndex = (tasks) => {
+  tasks.forEach( (val, index) => { val.index = index + 1 });
+};
+
+const editTask = (desc, index) => {
+  tasks[index].description = desc;
+  addToLocalStorage();
+};
+
+const displayTasks = () => {
+  taskWrapper.innerHTML = '';
+  taskWrapper.classList.add('drag-list');
+  const mylocal = getFromLocalStorage();
+  mylocal.forEach((tsk, index) => {
+    const li = document.createElement('li');
+    li.setAttribute('draggable', 'true');
+    li.setAttribute('data-index', index);
+    li.classList.add('draggable');
+    const checkbox = document.createElement('input');
+    checkbox.setAttribute('type', 'checkbox');
+    // eslint-disable-next-line no-unused-expressions
```

Comment:
I suggest you remove the `eslint-disable` command. Fixing errors will help you to follow **JS best practice** and you will get professional coding skill through fixing them